{"name": "Login with wrong password", "status": "passed", "description": "Signing in with the following credentials email : kamelmohsenkamel@gmail.com & password: wrong password", "attachments": [{"name": "test_login_failed_login", "source": "3e044b9c-ee44-4105-9b3d-11ed5cc6a03f-attachment.png", "type": "image/png"}], "start": 1583924363471, "stop": 1583924366077, "uuid": "fe8aa825-bc3e-4801-b080-8bbf12ff71ca", "historyId": "9002a5477b70c2d2732bd4879bd4c9b3", "testCaseId": "5849ff08927baae0bfe7bd1d291d6dac", "fullName": "Tests.login_tests.Test#test_case_2", "labels": [{"name": "severity", "value": "blocker"}, {"name": "parentSuite", "value": "End to End testing"}, {"name": "feature", "value": "Log in Tests"}, {"name": "suite", "value": "Login Tests"}, {"name": "subSuite", "value": "Login with wrong password"}, {"name": "story", "value": "Failing login"}, {"name": "tag", "value": "Do"}, {"name": "tag", "value": "Login"}, {"name": "host", "value": "DESKTOP-1RVKD4K"}, {"name": "thread", "value": "23264-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.login_tests"}]}